app/src/main/java/edu/uc/hornunjb/weatherwear/ui/main/MainViewModel.kt

```diff
@@ -11,9 +11,12 @@ class MainViewModel(private val repository: Repository) : ViewModel() {
 
     val myResponse: MutableLiveData<WeatherResponse> = MutableLiveData()
 
-    fun getPost() {
+    /*
+      Calls the getWeather method from the repository class
+     */
+    fun getWeather() {
```

Comment:
Same reasonings as above, the more important a method the more important the need for documentation and readability, in my opinion.